pal/src/cruntime/wchar.cpp

```diff
@@ -1482,9 +1482,8 @@ PAL_wcsstr(
         goto leave;
     }
 
-    if (*strCharSet == 0)
-    {
-        ret = (char16_t *)string;
+    if (*strCharSet == 0) {
```

Comment:
This looks like an unintentional change, the curly brace should start on the line after the `if`